src/utilities/openlab-helper.ts

```diff
@@ -0,0 +1,85 @@
+import axios from 'axios';
+import configurations from '../configurations';
+import * as _ from 'lodash';
+import * as Joi from '@hapi/joi';
+import 'joi-extract-type';
+import { isAxiosError } from './axios-helper';
+import logger from './logger';
+import WrappedError from '../exceptions/wrapped-error';
+import { OpenLabRedirectInfo } from '../features/courses/course-types';
+import * as qs from 'qs';
+import formHelper, { unmergeStrategies } from './form-helper';
+
+const openLabAxios = axios.create({
```

Comment:
I thought we talked about how this was a mistake on the renderer helper and should be in the class that way we could have multiple instances of the helper but use a default singleton?